lib/path.h

```diff
@@ -22,6 +22,7 @@
 //---------------------------------------------------------------------------
 
 #include "config.h"
+#include "settings.h"
```

Comment:
hmm.. I am not sure about this dependency.

Reply:
I don't like it either. The `Language` enum has to be moved from `settings.h` to some other header. `standards.h` seems way more appropriate - it's also a light header.

Reply:
Done